src/test/scala/chiselTests/LayerSpec.scala

```diff
@@ -176,22 +209,6 @@ class LayerSpec extends ChiselFlatSpec with Utils with MatchesAndOmits {
 
   }
 
-  they should "allow to define additional layer-colored probes using enables" in {
-
-    class Foo extends RawModule {
-      // Without this enable, this circuit is illegal because `C` is NOT enabled
-      // when `A` is enabled.  Cf. tests checking errors of this later.
-      layer.enable(A)
-      val a = IO(Output(Probe(Bool(), A)))
-      layer.block(C) {
-        val b = Wire(Bool())
-        define(a, ProbeValue(b))
-      }
-    }
```

Comment:
Great to remove this (and fix the checking) given that this emits with an illegal reference. This could be made to work, but we don't even have a way to describe this in the FIRRTL spec, i.e., there is no `layerblock A.C`.

(You know all this! I'm more leaving comments for bread crumbs if others come back to this later...)

Reply:
Breadcrumbs are great! Thanks! I'm unsure about this one, let's check!

I think the code in the test should not be supported as the probe target is not guaranteed to exist when `A` is enabled (`C` must be enabled, so the probe must at least have that color).

Additionally, enabling a layer should still allow sending out probes that aren't colored (not required to have color `A` [1].

This matches CIRCT's interpretation, AFAIK (FWIW).

I'm not sure what the mentioned `A.C` would be, and if it's some sort of sets-of-layers specification (?) not sure that changes anything here?